src/states/game/game.gd

```diff
@@ -15,7 +16,7 @@ const BLUE_CASTLE_DOOR = Vector2(480, 80)
 
 var enemy_moves: Array[Dictionary]
 var cards: Array[Resource]
-var num_rounds: int = 0
+var curr_round: int = 0
```

Comment:
Oh, it is lol. Remove the one in global

Reply:
waittttt this is different I think idk maybe

Reply:
`curr_rounds` is separate from `curr_stage`